assets/sass/scaffolding/_grid.scss

```diff
@@ -13,8 +13,12 @@ $flexGridSpacing: $spacingUnit;
  */
 .flex-grid {
     @include reset-list();
-    // Overrides constrained content containers
-    max-width: 100% !important;
+
+    .u-constrained-content-wide & {
+        // Overrides constrained content containers
+        // eg. grids within flexible content areas
+        max-width: 100% !important;
```

Comment:
In my efforts to fix this bug:

![image](https://user-images.githubusercontent.com/394376/67089067-a95bc000-f19e-11e9-9790-54fe9cde9cac.png)

... I caused this one: 

![image](https://user-images.githubusercontent.com/394376/67089090-b678af00-f19e-11e9-9135-97bed97f0476.png)

Reply:
This feels…very specific?

Reply:
I'm not really sure how else to achieve it – it's because the `s-prose` classes do things to `ul` and `li` elements which are quite difficult to override. Suggestions welcome!

Reply:
Don't put these inside an s-prose?

Reply:
As discussed on chat, this all makes sense for now, and we can come back to it to try and find a way to not make the prose class a requirement for a flexible content area.